### DISCUSSION TITLE
Thêm comment trực tiếp trên trang blog

### DISCUSSION THREAD

**Discussion by trongnguyen24** [sveltevietnam/sveltevietnam.dev#211] [Ideas]
Mình nghĩ việc cho phép mọi người comment thẳng trên blog giúp mọi người vào bình luận thoải mái, tạo thêm tương tác cho blog.

có thể dùng Utterances, cái tool chạy qua GitHub, cho đồng bộ với bài viết luôn.

**Comment by vnphanquang** [comment]
@trongnguyen24 cảm ơn bạn. Vấn đề này mình đã suy nghĩ đến và sẽ cân nhắc. Việc thêm comment sẽ phát sinh khá nhiều vấn đề cồng kềnh nhưng chưa biết có thật sự mang lại giá trị và người dùng có nhu cầu ko. Có thể mình sẽ khảo sát người dùng trước.
